package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "@prefecthq/eslint-config": "^1.0.31",
+    "@testing-library/dom": "^9.3.3",
```

Comment:
I added this at one point to simulate popstate events. That's no longer needed in this PR, but could be coming? I'll let you decide if this should be removed

Reply:
If it's not used in this PR let's remove it for now.